helpers.js

```diff
@@ -321,6 +321,31 @@ const installAllPackagesToExistingProject = async () => {
 
 exports.installAllPackagesToExistingProject = installAllPackagesToExistingProject
 
+
+const insert = async (fileToInsertInto, whatToInsert, lineToInsertAt) => {
```

Comment:
Maybe this should be an object? I think it could use another option, line to insert after to pattern match the file for a string.